fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/services/ReplacePropertiesService.java

```diff
@@ -31,12 +31,14 @@
    * Replace the properties of this object with the properties from the given
    * model
    *
+   * @param tx the Transaction
    * @param fedoraResource the Fedora resource to update
-   * @param inputModel the input model
-   * @param originalTriples the original triples
+   * @param requestBodyStream the body of the request
+   * @param contentType the original triples
```

Comment:
update this param description to match the field

Reply:
This has been fixed.